src/Extend/ModelVisibility.php

```diff
@@ -44,7 +44,7 @@ public function __construct(string $modelClass)
     {
         $this->modelClass = $modelClass;
 
-        if (! method_exists($modelClass, 'registerVisibilityScoper')) {
+        if (class_exists($this->modelClass) && ! is_callable([$modelClass, 'registerVisibilityScoper'])) {
```

Comment:
Should this fail silently if the model doesn't exist? Although I suppose that would be inline with behavior of current extensions.

Reply:
I think the original idea behind the policy was to apply only when flags was installed. I don't think we need to log anything here.